Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -2426,6 +2465,58 @@ AODProducerWorkflowDPL::TrackExtraInfo AODProducerWorkflowDPL::processBarrelTrac
   return extraInfoHolder;
 }
 
+AODProducerWorkflowDPL::TrackQA AODProducerWorkflowDPL::processBarrelTrackQA(int collisionID, std::uint64_t collisionBC, GIndex trackIndex,
+                                                                             const o2::globaltracking::RecoContainer& data, const std::map<uint64_t, int>& bcsMap)
+{
+  TrackQA trackQAHolder;
+  auto contributorsGID = data.getSingleDetectorRefs(trackIndex);
```

Comment:
As I wrote previously, you can use simply 
```
auto tpcGID = data.getTPCContributorGID(trackIndex);
if (tpcGID.isIndexSet()) {
 const auto& tpcOrig = data.getTPCTrack(tpcGID);
 ...
```
getting only TPC index is much cheaper than building full contributorsGID set.

Reply:
I need a moment of combined track. If there is another way, you can suggest it.

Reply:
Fixed: https://github.com/AliceO2Group/AliceO2/pull/12387/commits/22458c094dcaa86e3ac5cf9d9236451c892532aa